database/database.go

```diff
@@ -182,6 +183,18 @@ func seed() {
 		},
 	}
 
+	OAuths := []models.OAuth{
+		{
+			UserID:       1,
+			Name:         "USw integration",
+			Description:  "Just some integration",
+			Scopes:       []string{"style", "user"},
+			ClientID:     "publicccc_client",
+			ClientSecret: "secreettUwU",
+			RedirectURI:  "https://gusted.xyz/callback_helper/usw",
+		},
+	}
+
```

Comment:
Make this only available in dev environment?

Reply:
Isn't it already? It will only be seeded once you set DB_DROP env

Reply:
Oh yeah, I forgot. 😅